tests/Avalonia.Animation.UnitTests/TransitionsTest.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Avalonia.Animation;
+using Avalonia.Controls;
+using Avalonia.Styling;
+using Avalonia.UnitTests;
+using Avalonia.Data;
+using Xunit;
+using Avalonia.Animation.Easings;
+
+namespace Avalonia.Animation.UnitTests
+{
+    public class TransitionsTest
```

Comment:
Nit: all other unit test classes are named with the plural, so `TransitionsTests`.